### ISSUE TITLE
Note durations in a midi file increase after loading and dumping via symusic.Score

### ISSUE THREAD

**Issue by pstrepetov** [Yikai-Liao/symusic#48]
After loading and dumping certain midi file its duration increased

Python code:

```
import symusic

score = symusic.Score.from_file("orig.mid")
score.dump_midi("midi_changed.mid")
```

I've compared MIDI files with html midi player (https://cifkao.github.io/html-midi-player/) and expected the same output, but midi are not the same.

Here are screenshots of midi visualization in the midi player:
Original midi:
![original](https://github.com/user-attachments/assets/994a5e45-1260-4668-8405-116ab5622b2a)

MIDI after loading with symusic
![after loading with symusic](https://github.com/user-attachments/assets/f8805502-15f6-4bd1-83ed-6a17bc214ca7)

[Link to folder with MIDI files](https://drive.google.com/drive/folders/16eUxPoP8uvDRydzi9N1-k_bc9IAdpKLZ?usp=sharing)

symusic version: 0.4.8
python: 3.10.14

**Comment by lzqlzzq** [comment]
Seems to be a problem with tempo change, we are working on it.
![0aR5A7Fgsj](https://github.com/user-attachments/assets/1a6771db-e828-4f1e-9060-3d8e8fe8143f)
3 tempo changes in `orig.mid`
![Xn9Qi24ru8](https://github.com/user-attachments/assets/1e6e35c8-7c7d-4195-8cc5-71d1b6a1c409)
7 tempo changes parsed by `symusic`

**Comment by Yikai-Liao** [comment]
Problem has been fixed in the `0.5.0`. I finally release this version with lots of changes.

**Comment by pstrepetov** [comment]
Thank you so much for the quick response!
